CRM/Twingle/Submission.php

```diff
@@ -535,8 +541,9 @@ public static function createLineItems($values, $submission, $profile): array {
         $line_item_data['price_field_id'] = $price_field->price_field_id;
         $line_item_data['description'] = $price_field->description;
       }
-      // If the product ID is -22, it's an additional donation. See #102
-      elseif ($product['id'] === -22) {
+      // Identify additional donations by their ID.
+      // See https://github.com/systopia/de.systopia.twingle/issues/102
+      elseif ($product['id'] === self::TWINGLE_ADDITIONAL_DONATION_ID) {
```

Comment:
Perfect! Just nitpicking: this could preferably be a *yoda condition*.

Reply:
Hurt my brain, the Yoda condition coding style does.